sensors/shtc3Sensor/SHTC3Sensor.cpp

```
@@ -0,0 +1,146 @@
+        std::cout << "Invalid measurement interval: " << interval_ms << "ms, clamping to minimum: " << kMinMeasurementIntervalMs << "ms" << std::endl;
+        measurement_interval_ms_ = kMinMeasurementIntervalMs;
+    } else {
+        measurement_interval_ms_ = interval_ms;
+    }
+}
+
+void SHTC3Sensor::setMeasurementCallback(void (*callback)(int32_t temperature, int32_t humidity)) {
+    measurement_callback_ = callback;
+}
+
+bool SHTC3Sensor::startContinuousMeasurement() {
```

Clang (14.0.6)

[warning] use a trailing return type for this function

(modernize-use-trailing-return-type)